src/entities/Proposal/types.ts

```diff
@@ -63,6 +63,7 @@ export type ProposalAttributes<C extends Record<string, unknown> = any> = {
   created_at: Date
   updated_at: Date
   textsearch: SQLStatement | string | null | undefined
+  coauthors?: string[] | null
```

Comment:
I'm not sure if we should add this field here if it's not going to be a column in the db.

Perhaps you could just change the return type of the methods in the model to

`ProposalAttributes & {coauthors?: string[] | null}`